### PR TITLE
nautilus: common:  ignore SIGHUP prior to fork

### PR DESCRIPTION
**Pull request by smithfarm**
backport tracker: https://tracker.ceph.com/issues/46592

---

backport of

* https://github.com/ceph/ceph/pull/35844
* #36194

parent tracker: https://tracker.ceph.com/issues/46269

this backport was staged using ceph-backport.sh version 15.1.1.389
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

---

To Do:

- [x] https://github.com/ceph/ceph/pull/36194 merged into master
- [x] https://github.com/ceph/ceph/pull/36194 cherry-picked into this PR

### CODE REVIEW

**Comment by batrick** [comment]
Follow-up fix to add: https://github.com/ceph/ceph/pull/36194

**Comment by batrick** [comment]
@smithfarm why DNM?

**Comment by smithfarm** [comment]
@batrick Right, just needed a prod. Thanks!

**Comment by smithfarm** [comment]
@batrick #36195 is the octopus backport. It's still open...

**Comment by batrick** [comment]
> @batrick #36195 is the octopus backport. It's still open...

Do you mean we don't merge the nautilus backport until the octopus one is merged to prevent upgrade regressions? (I like that policy but I don't think we actually do it?)

**Comment by smithfarm** [comment]
> Do you mean we don't merge the nautilus backport until the octopus one is merged to prevent upgrade regressions? (I like that policy but I don't think we actually do it?)

No, that's not (exactly) what I meant, because - as you pointed out - we don't have such a policy. But it's true that the leads intentionally delay some backports that are deemed risky. This is sometimes called "letting it bake in foo" (where foo might be "master" or a newer release cycle like octopus).

**Comment by theanalyst** [comment]
Marking for testing as the octopus pr is in

**Comment by yuriw** [comment]
[wip-yuri-testing-2020-09-08-1946-nautilus](https://trello.com/c/IzomWqYd/1004-wip-yuri-testing-2020-09-08-1946-nautilus)

**Review by batrick — APPROVED** [review]

**Pull request merged**
